awx/main/tasks.py

```diff
@@ -692,9 +698,12 @@ class BaseTask(object):
     model = None
     event_model = None
     abstract = True
-    cleanup_paths = []
     proot_show_paths = []
 
+    def __init__(self, *args, **kwargs):
+        super(BaseTask, self).__init__(*args, **kwargs)
```

Comment:
Do you need this?  The superclass is just `object`.

Reply:
...since this is `BaseTask`, not some derived class, I agree strongly. Will push after re-runs of tests.